MANUAL.html

```diff
@@ -31,22 +31,22 @@ <h1 id="rclone-syncs-your-files-to-cloud-storage">Rclone syncs your files to clo
 <li><a href="https://rclone.org/donate/">Donate.</a></li>
 </ul>
 <h2 id="about">About rclone</h2>
-<p>Rclone is a command line program to manage files on cloud storage. It is a feature rich alternative to cloud vendors' web storage interfaces. <a href="#providers">Over 40 cloud storage products</a> support rclone including S3 object stores, business &amp; consumer file storage services, as well as standard transfer protocols.</p>
+<p>Rclone is a command-line program to manage files on cloud storage. It is a feature-rich alternative to cloud vendors' web storage interfaces. <a href="#providers">Over 40 cloud storage products</a> support rclone including S3 object stores, business &amp; consumer file storage services, as well as standard transfer protocols.</p>
```

Comment:
This file is auto-generated from many markdown files, and comment-based documentation in source. So the changes must be done in the correct sources, and this file removed from the PR.
This specific line has been fixed in correct source, docs/content/_index.md, but I haven't gone through to verify if this is the case for all of the changes.